src/main/java/com/trueffect/sql/crud/JobCrud.java

```diff
@@ -11,13 +11,13 @@
 public class JobCrud {
     public static Job getJobOf(int id)throws Exception {
             Job job=null;
-            PostgresSQLConnection.connectionDB();
+            DatabasePostgres.getConection();
             try {
                           String sql = "SELECT id, name_job\n" +
```

Comment:
Please align the query using the format RIVER .Use as example this url :http://www.sqlstyle.guide/#white-space